tests/unit/test_docs.py

```diff
@@ -87,5 +113,7 @@ def test_folders_contain_index(data):
 
 @pytest.mark.parametrize("start_end", START_END_LINES, ids=[section["name"] for section in START_END_LINES])
 def test_docs_start_end_lines(start_end):
-    assert README_LIST[start_end["start_line"]] == start_end["start_value"]
-    assert README_LIST[start_end["end_line"]] == start_end["end_value"]
+    start_line_value = _get_start_line(start_end["name"])
+    end_line_value = _get_end_line(start_end["name"])
+    assert README_LIST[start_line_value] == start_end["start_value"]
+    assert README_LIST[end_line_value - 1] == start_end["end_value"]
```

Comment:
Any reason we can't make this more try?  Perhaps by just passing in a arg of "start" or "end"?  Then have some constants for the regex patterns?

Reply:
After looking at this a little I'm not sure I know what you mean. Let's meet tomorrow to discuss quickly.